vimeo-networking/src/main/java/com/vimeo/networking/model/Connection.java

```diff
@@ -57,12 +57,6 @@
     @GsonAdapterKey("viewable_total")
     public int viewableTotal;
 
-    @GsonAdapterKey("new_total")
-    int newTotal;
-
-    @GsonAdapterKey("unread_total")
-    int unreadTotal;
-
```

Comment:
Nice to be able to remove these from Connection since they aren't applicable anywhere but in Notifications